README.md

```diff
@@ -4,32 +4,43 @@ __URLs__
 
 * Name all of the parts of the url that you can remember.  In your own words describe what they do.
 * Name the pieces of the following urls:
-	* `https://www.google.com/`
-	* `https://workbook.galvanize.com/cohorts/41/learning_experiences/367`
-	* `http://locahost:5000/animals/puppies?onlycute=1&size=medium#firstpuppy`
-	* `https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_Error`
+	* `https://www.google.com/`-->protocol, domain
+	* `https://workbook.galvanize.com/cohorts/41/learning_experiences/367` -->protocol, domain, path
+	* `http://locahost:5000/animals/puppies?onlycute=1&size=medium#firstpuppy` -->protocol, domain, port, path, query string, anchor tag
+	* `https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_Error` -->protocol, domain, path, anchor tag
 * Can a server use more than 1 port?
+```Yes```
 * Why is https different than http?
+```it is the secure version of http```
 * How does a server interpret the following url's query paramter.  What data structure does it create on the server?
 
 ```
 http://locahost:5000/animals?puppies=fido&puppies=max&puppies=moxie
 ```
 
+```I think that an object is created, more specifically, perhaps, an array```
```

Comment:
Good guess, 

I think it would usually just set `pupppies` equal to the last value `moxie`